internal/node/node_patches.js

```diff
@@ -9,12 +9,18 @@ var fs$1 = _interopDefault(require('fs'));
 
 var commonjsGlobal = typeof globalThis !== 'undefined' ? globalThis : typeof window !== 'undefined' ? window : typeof global !== 'undefined' ? global : typeof self !== 'undefined' ? self : {};
 
-function unwrapExports (x) {
```

Comment:
This one was interesting. Seems to just be changes what rollup outputs, but interesting that the `unwrapExports` stuff is all gone, and I guess `createCommonjsModule` has a completely new implementation.

Reply:
yeah I think we don't really care - in fact there's a mechanism to get GitHub to stop showing us this generated file which I'd like to look into sometime https://github.com/github/linguist#generated-code
(it excludes from the lines count but I dunno about code review and source browsing...)